### PR TITLE
add paddle.static.data

### PR DESCRIPTION
**Pull request by ForFishes**
### PR types

New features
### PR changes

APIs
### Describe

Except the `dtype` is different in default parameter settings, It is similar as `fluid.data` api. When `dtype` is not set, the dtype will get from the global dtype by `paddle.get_default_dtype()`.

### CODE REVIEW

**Review by XiaoguangHu01 — DISMISSED** [review]

File: python/paddle/static/input.py [hunk 1]
```diff
@@ -47,8 +48,79 @@ def __init__(self, shape=None, dtype='float32', name=None):
...
+
+def data(name, shape, dtype=None, lod_level=0):
+    """
+    **Data Layer**
+
+    This function creates a variable on the global block. The global variable
+    can be accessed by all the following operators in the graph. The variable
+    is a placeholder that could be fed with input, such as Executor can feed
+    input into the variable. Except the `dtype` is different in default parameter 
+    settings, It is similar as `fluid.data` api. When `dtype` is not set, the dtype
```
Comment (jzhang533) [inline python/paddle/static/input.py]: 这个api的文档里就不需要提fluid了

File: python/paddle/static/input.py [hunk 2]
```diff
@@ -47,8 +48,79 @@ def __init__(self, shape=None, dtype='float32', name=None):
...
+
+    Args:
+       name (str): The name/alias of the variable, see :ref:`api_guide_Name`
+           for more details.
+       shape (list|tuple): List|Tuple of integers declaring the shape. You can
+           set "None" or -1 at a dimension to indicate the dimension can be of any
+           size. For example, it is useful to set changeable batch size as "None" or -1.
+       dtype (np.dtype|VarType|str, optional): The type of the data. Supported
+           dtype: bool, float16, float32, float64, int8, int16, int32, int64,
+           uint8. Default: float32. When `dtype` is not set, the dtype will get
```
Comment (jzhang533) [inline python/paddle/static/input.py]: default是none

File: python/paddle/static/input.py [hunk 3]
```diff
@@ -47,8 +48,79 @@ def __init__(self, shape=None, dtype='float32', name=None):
...
+           use LoD level, see :ref:`user_guide_lod_tensor` . Default: 0.
+
+    Returns:
+        Variable: The global variable that gives access to the data.
+
+    Examples:
+        .. code-block:: python
+
+          import numpy as np
+          import paddle.fluid as fluid
```
Comment (jzhang533) [inline python/paddle/static/input.py:57]: executor这些还在fluid下是吧？

File: python/paddle/static/input.py [hunk 4]
```diff
@@ -47,8 +48,79 @@ def __init__(self, shape=None, dtype='float32', name=None):
...
+    settings, It is similar as `fluid.data` api. When `dtype` is not set, the dtype
+    will get from the global dtype by `paddle.get_default_dtype()`.
+
+    Args:
+       name (str): The name/alias of the variable, see :ref:`api_guide_Name`
+           for more details.
+       shape (list|tuple): List|Tuple of integers declaring the shape. You can
+           set "None" or -1 at a dimension to indicate the dimension can be of any
+           size. For example, it is useful to set changeable batch size as "None" or -1.
+       dtype (np.dtype|VarType|str, optional): The type of the data. Supported
```
Comment (jzhang533) [inline python/paddle/static/input.py]: 不需要暴露vartype

File: python/paddle/static/input.py (hunk 1 above)
Comment (ForFishes) [reply python/paddle/static/input.py]: ok, fix it

File: python/paddle/static/input.py (hunk 2 above)
Comment (ForFishes) [reply python/paddle/static/input.py]: ok, fix it

File: python/paddle/static/input.py (hunk 3 above)
Comment (ForFishes) [reply python/paddle/static/input.py:57]: 对的，这不还没移动

File: python/paddle/static/input.py (hunk 4 above)
Comment (ForFishes) [reply python/paddle/static/input.py]: ok，fix it

**Review by jzhang533 — APPROVED** [review]
lgtm

**Review by XiaoguangHu01 — APPROVED** [review]
LGTM

**Pull request merged**